### PR TITLE
Fixed v.mod file

### PR DESCRIPTION
**Pull request by AhmedsAxioms**
There were a few new additions to the v.mod file that the examples didn't point out so I thought to add it.

i  :
* Changed the casing for the module in the example to Snake_case
* Added new parts that's generated by `v new`.

### CODE REVIEW

File: doc/docs.md [hunk 1]
```diff
@@ -3545,9 +3545,11 @@ created your module with `v new` you already have v.mod file). For
 example:
 ```v ignore
 Module {
-	name: 'mymodule',
-	description: 'My nice module wraps a simple C library.',
+	name: 'my_module'
+	description: 'This is my_module's description.'
 	version: '0.0.1'
+	license: ''
```
Comment (Gladear) [inline doc/docs.md:3551]: Since this is an example, can you add an example of license ? MIT maybe ? Since this is the license V is licensed under

Comment (AhmedsAxioms) [reply doc/docs.md:3551]: oh yeah will do!, thanks.

File: doc/docs.md [hunk 2]
```diff
@@ -3545,9 +3545,11 @@ created your module with `v new` you already have v.mod file). For
 example:
 ```v ignore
 Module {
-	name: 'mymodule',
-	description: 'My nice module wraps a simple C library.',
+	name: 'my_module'
+	description: 'This is my_module's description.'
```
Comment (spytheman) [inline doc/docs.md:3549]: this is an invalid description

File: doc/docs.md [hunk 3]
```diff
@@ -3545,9 +3545,11 @@ created your module with `v new` you already have v.mod file). For
...
 	version: '0.0.1'
+	license: 'MIT'
+	repo_url: 'https://github.com/Example/ExampleRepo'
 	dependencies: []
```
Comment (spytheman) [inline doc/docs.md:3553]: these are not required, and docs.md is not the place for that imho

File: doc/docs.md (hunk 2 above)
Comment (JalonSolov) [reply doc/docs.md:3549]: If you have a single quote in the string, double quotes are needed to enclose.

**Pull request closed without merging**